authentication/authentication-service/src/main/java/io/mosip/authentication/service/kyc/controller/KycAuthController.java

```diff
@@ -189,6 +189,7 @@ public EKycAuthResponseDTO processKyc(@Validated @RequestBody EkycAuthRequestDTO
 				kycReqValidator.validateIdvId(ekycAuthRequestDTO.getIndividualId(), idType, errors);
 				if(AuthTypeUtil.isBio(ekycAuthRequestDTO)) {
 					kycReqValidator.validateDeviceDetails(ekycAuthRequestDTO, errors);
+					kycReqValidator.validateAge(ekycAuthRequestDTO, errors);
```

Comment:
why the age validation added inside Bio auth type?

Reply:
took that validation outside. Please review.